src/AAVE/Modal/BorrowModal.jsx

```diff
@@ -122,7 +122,7 @@ State.init({
 });
 
 function updateGas() {
-  if (["ETH", "WETH"].includes(symbol)) {
+  if (symbol === config.nativeCurrency.symbol) {
```

Comment:
If symbol is `WETH` or `WMATIC`, we don't need to call `borrowETHGas()` now? Why the logic has changed?

Reply:
See #16 reviews. 
Before that, we only needed one token, two layers of check for fallback processing.
Now we need to handle the token precisely because the wrap token should perform an erc20 function